### PR TITLE
Civ 3120 claimstore client

### PR DESCRIPTION
**Pull request by TatsyMaslova**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/CIV-3120

### Change description ###

client to connect to claim store to retrieve ocmc claims for citizen ui

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

**Review by laarmada — APPROVED** [review]

File: src/integrationTest/java/uk/gov/hmcts/reform/civil/controllers/cases/CasesControllerTest.java [hunk 1]
```diff
@@ -34,6 +39,24 @@ public class CasesControllerTest extends BaseIntegrationTest {
         + "\n"
         + " }\n"
         + "}";
+    private static final String CLAIMANT_CLAIMS_URL = "/cases//claimant/{submitterId}";
```
Comment (GorilaNaranja) [inline src/integrationTest/java/uk/gov/hmcts/reform/civil/controllers/cases/CasesControllerTest.java:42]: should be `/cases/claimant/{submitterId`?

Comment (TatsyMaslova) [reply src/integrationTest/java/uk/gov/hmcts/reform/civil/controllers/cases/CasesControllerTest.java]: done 👍

**Review by GorilaNaranja — APPROVED** [review]

**Review by taomoh — APPROVED** [review]

File: charts/civil-service/values.yaml [hunk 2]
```diff
@@ -43,6 +43,7 @@ java:
     DOCMOSIS_TORNADO_URL: https://docmosis.{{ .Values.global.environment }}.platform.hmcts.net
     DOCUMENT_MANAGEMENT_URL: http://dm-store-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
     CASE_DOCUMENT_AM_URL: http://ccd-case-document-am-api-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
+    CLAIM_STORE_URL: http://cmc-claim-store-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
```
Comment (mounikahmcts) [inline charts/civil-service/values.yaml:46]: You might need to add same env variable  in https://github.com/hmcts/civil-citizen-ui/blob/master/charts/civil-citizen-ui/values.preview.template.yaml#L85.

**Review by mounikahmcts — CHANGES_REQUESTED** [review]

File: charts/civil-service/values.yaml (hunk 2 above)
Comment (TatsyMaslova) [reply charts/civil-service/values.yaml:46]: yes, that would be done in the corresponding story https://tools.hmcts.net/jira/browse/CIV-2747 in the citizen ui branch

**Review by mounikahmcts — APPROVED** [review]

**Pull request merged**